angular-bind-html-compile.js

```diff
@@ -7,14 +7,19 @@
         return {
             restrict: 'A',
             link: function (scope, element, attrs) {
-                scope.$watch(function () {
+                scope.$watch(function() {
```

Comment:
Ah sorry to be a code style stickler, but can we keep the space here after the `function` keyword?